core/trace/dt_span_exporter.go

```diff
@@ -138,9 +125,44 @@ func (e *dtSpanExporterImpl) export(ctx context.Context, t exportType, spans dtS
 		return errors.New("unexpected response code: " + strconv.Itoa(resp.StatusCode))
 	}
 
+	e.logger.Debug("Export successful")
```

Comment:
Could log exact HTTP code & reason phrase

Reply:
Actually there's already some code in `performHttpRequest` that does this, will remove this log line.